mite_selenium/__init__.py

```diff
@@ -243,6 +243,12 @@ def _start(self):
         logger.debug(f"Selenium-wire machine IP: {addr}")
         self._context.raw_webdriver = self._remote
 
+    def wait_for_request(self, url, timeout=5):
+        return self._context.raw_webdriver.wait_for_request(url, timeout)
+
+    def get_requests(self):
+        return self._context.raw_webdriver.requests
+
```

Comment:
could you use `self._remote` here ? I'ts shorter than `self._context.raw_webdriver` and looks like the same thing ?

Reply:
Done 👍